tests/test_workflows.py

```diff
@@ -791,22 +791,22 @@ def test_get_xfa(url, name):
     ("url", "name", "strict"),
     [
         (
-            "https://corpora.tika.apache.org/base/docs/govdocs1/988/988698.pdf",
+            "https://github.com/user-attachments/files/18381765/tika-988698.pdf",
             "tika-988698.pdf",
             False,
         ),
         (
-            "https://corpora.tika.apache.org/base/docs/govdocs1/914/914133.pdf",
+            "https://github.com/user-attachments/files/18382162/914133.pdf",
             "tika-988698.pdf",
```

Comment:
there is here an error out of the links but that should be fixed
```suggestion
            "tika-914133.pdf",
```